alf/environments/suite_socialbot.py

```diff
@@ -91,6 +94,10 @@ def load(environment_name,
     Returns:
         A PyEnvironmentBase instance.
     """
+    global _unwrapped_env_in_process_
+    assert not _unwrapped_env_in_process_, \
+        "You cannot create more envs once there has been an env in the main process!"
+    _unwrapped_env_in_process_ |= not wrap_with_process
```

Comment:
One way to reduce code duplication is to make a class for this. We declare a global object g_wrap_checker for each load module. And calling g_wrap_checker.check() inside the load function.

Reply:
Changed.